HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt

```diff
@@ -152,6 +153,9 @@ class MainActivity : AppCompatActivity() {
                     if (error != null) {
                         addToLogView(placementName + "(HeliumInterstitialAd) didShow failed with error: " + error.getMessage())
                     } else {
+                        Handler(Looper.getMainLooper()).post {
+                            binding.btnLoad.isEnabled = true
```

Comment:
I'm actually not a fan of blocking load buttons in general.

Reply:
Can make new adjustments here as well.